requirements/common.in

```diff
@@ -30,7 +30,7 @@ SQLAlchemy
 argon2-cffi
 
 # Needed for S3 file uploads
-boto
+boto3==1.9.240
```

Comment:
Just `boto3`; don’t pin a specific version here (especially one that causes downgrades).